app/views/pages/support.html.erb

```diff
@@ -26,6 +26,11 @@
         For questions about a dataset, you can contact the publisher directly if they’ve provided contact details on the
```

Comment:
The line above this talks about a 'data request option' - but I can't see one on the page. Is it somewhere else?

Reply:
Second option in form at https://data.gov.uk/support

Reply:
OK. It's a bit strange as I'm reading top-to-bottom and there's nothing in the text that talks about the form explicitly. Maybe we could add 'below' as a pointer?

Reply:
Added